
# Content-related messages

content.content_missing=O conte\ufffddo n\ufffd est\ufffd ausente: \n n\ufffd: {0} \n leitor: {1} \n Entre em contato com o administrador do sistema.
content.runtime_exec.property_moved=A propriedade ''errorCodes'' movida para baixo para a classe RuntimeExec

index.recovery.out_of_date=Os \ufffdndices n\ufffdo s\ufffdo sincronizadas com o banco de dados.
index.tracking.starting=\ufffdndice de recupera\ufffd\ufffdo iniciados.
index.tracking.complete=Recupera\ufffd\ufffdo dos \ufffdndices conclu\ufffdda.
index.tracking.progress=\tProcessing opera\ufffd\ufffdes em torno de {0}.
index.recovery.starting=\ufffdndice de recupera\ufffd\ufffdo iniciada: {0} transa\ufffd\ufffdes.
index.recovery.complete=Recupera\ufffd\ufffdo dos \ufffdndices conclu\ufffdda.
index.recovery.progress=\t{0} % completo.
index.recovery.terminated=\ufffdndice de recupera\ufffd\ufffdo encerrado.

node.archive.msg.busy=A remo\ufffd\ufffdo a granel ou de restaura\ufffd\ufffdo est\ufffd em andamento

content.http_reader.err.no_connection=N\ufffdo \ufffd poss\ufffdvel conectar ao servidor Alfresco remoto via HTTP: {0}
content.http_reader.err.no_authentication=O leitor HTTP foi incapaz de se autenticar no servidor remoto: {0} \n
content.http_reader.err.check_cluster=Certifique-se que "replicateUpdates' e 'replicateUpdatesViaCopy' est\ufffd habilitado para org.alfresco.cache.ticketsCache' cache. Verifique se a configura\ufffd\ufffdo do cluster geral est\ufffd correto e funcionando.
content.http_reader.err.unrecognizedUm erro desconhecido ocorreu ao tentar baixar o conte\ufffddo do servidor remoto: \n Servidor: {0} Content \n: {1} \n HTTP Resposta: {2}

metadata.extraction.err.type_conversion=Extra\ufffd\ufffdo de metadados falhou porque um valor extra\ufffddo Falha ao converter para o tipo necess\ufffdrio: \n Extractor: {0} \n Target Propriedade QName: {1} \n Obrigat\ufffdrio Tipo: {2} \n Extra\ufffddo Valor: {3}

transform.err.format_or_password=Falha ao converter conte\ufffddo, possivelmente devido a um arquivo formatado incorretamente ou protegido por senha.

content.routing.err.invalid_default_store=O 'defaultStoreName',''{0}''n\ufffdo se refere a uma loja em 'storesByName "({1}).
